### ISSUE TITLE
Uable to setup LDAP

### ISSUE THREAD

**Issue by Ryonez** [jellyfin/jellyfin#2021]
**Describe the bug**

I'm unable to setup LDAP authentication. After entering the information needed into the plugins config, clicking save productions a spinning loading circle forever.

**To Reproduce**

1. Install LDAP plugin V4
2. Go to the plugins settings and fill it out.
3. Click save
4. See endlessly spinning circle.

**Expected behavior**

For it to work and save the configuration

**Logs**

```
2019-11-18 12:05:29,552 DEBG 'jellyfin' stdout output:
[12:05:29] [ERR] Error processing request
System.FormatException: Guid should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
at System.Guid.GuidResult.SetFailure(ParseFailureKind failure, String failureMessageID, Object failureMessageFormatArgument, String failureArgumentName, Exception innerException)
at System.Guid.TryParseGuidWithNoStyle(ReadOnlySpan`1 guidString, GuidResult& result)
at System.Guid.TryParseGuid(ReadOnlySpan`1 guidString, GuidStyles flags, GuidResult& result)
at System.Guid..ctor(String g)
at MediaBrowser.Api.PluginService.Post(UpdatePluginConfiguration request) in /tmp/pkgbuild-0/jellyfin/src/jellyfin-10.4.1/MediaBrowser.Api/PluginService.cs:line 251
at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task) in /tmp/pkgbuild-0/jellyfin/src/jellyfin-10.4.1/Emby.Server.Implementations/Services/ServiceExec.cs:line 113
at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken) in /tmp/pkgbuild-0/jellyfin/src/jellyfin-10.4.1/Emby.Server.Implementations/Services/ServiceHandler.cs:line 80
at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken) in /tmp/pkgbuild-0/jellyfin/src/jellyfin-10.4.1/Emby.Server.Implementations/HttpServer/HttpListenerHost.cs:line 519
```

**Screenshots**

**System (please complete the following information):**
 - OS: Docker (on unRaid)
 - Browser: Firefox V70.0.1 64 bit
 - Jellyfin Version: 10.4.1
 - Reverse proxy: nginx (letsencript docker by linuxiso)

**Additional context**

Atempted configuration:
![image](https://user-images.githubusercontent.com/16157130/69015808-d0571d00-09fc-11ea-9883-eae714dddbe7.png)

**Comment by Ryonez** [comment]
Is this being looked at? I'm not sure what the regression label mean in terms of resolving the issue.

**Comment by Bond-009** [comment]
Should be fixed by #2008
